### PR TITLE
Add jackson and vm extensions, fixes #306 and #318

### LINKED ISSUE #306: Create a Camel Json Jackson extension
https://github.com/apache/camel/tree/master/components/camel-jackson
https://github.com/quarkusio/quarkus/tree/master/extensions/jackson

### PR DESCRIPTION
**Pull request by gnodet**

### CODE REVIEW

File: extensions/jackson/runtime/pom.xml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        <artifactId>camel-quarkus-jackson-parent</artifactId>
+        <version>0.2.1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>camel-quarkus-jackson</artifactId>
+    <name>Camel Quarkus :: Jackson :: Runtime</name>
+
+    <properties>
+        <firstVersion>0.2.0</firstVersion>
```
Comment (davsclaus) [inline extensions/jackson/runtime/pom.xml]: Its 0.2.1 or if we decide to call it 0.3.0 (the next release)

File: extensions/jackson/runtime/pom.xml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                  <artifactId>jackson-databind</artifactId>
+               </exclusion>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.module</groupId>
+                    <artifactId>jackson-module-jaxb-annotations</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
```
Comment (davsclaus) [inline extensions/jackson/runtime/pom.xml]: Do we need this dependency?

File: extensions/vm/runtime/pom.xml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        <artifactId>camel-quarkus-vm-parent</artifactId>
+        <version>0.2.1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>camel-quarkus-vm</artifactId>
+    <name>Camel Quarkus :: VM :: Runtime</name>
+
+    <properties>
+        <firstVersion>0.2.0</firstVersion>
```
Comment (davsclaus) [inline extensions/vm/runtime/pom.xml]: Its 0.2.1 or if we decide to call it 0.3.0 (the next release)

**Comment by lburgazzoli** [comment]
ok to test

**Pull request merged**